modules/microshift-install-bootc-conc.adoc

```diff
@@ -6,14 +6,14 @@
 [id="microshift-bootc-conc_{context}"]
 = About image mode for {op-system-base-full}
 
-Image mode for {op-system-base-full} is a Technology Preview deployment method that uses a container-native approach to build, deploy, and manage the operating system as a bootc image. By using bootc, you can build, deploy, and manage the operating system as if it is any other container.
+By using image mode for {op-system-base-full} {op-system-base}, you can administer the operating system just as you would any other container. Image mode for {op-system-base} is a deployment method that uses a container-native approach to build, deploy, and manage the operating system as a bootc image.
 
 * This container image uses standard OCI or Docker containers as a transport and delivery format for base operating system updates.
 * A bootc image includes a Linux kernel that is used to start the operating system.
 * By using bootc containers, developers, operations administrators, and solution providers can all use the same container-native tools and techniques.
 
-Image mode splits the creation and installation of software changes into two steps: one on a build system and one on a running target system.
+Image mode for {op-system-base} splits the creation and installation of software changes into two steps: one on a build system and one on a running target system.
 
-* In the build-system step, a Podman build inspects the RPM files available for installation, determines any dependencies, and creates an ordered list of chained steps to complete, with the end result being a new operating system available to install.
+* In the build-system step, a Podman build inspects the RPM files available for installation, determines any dependencies, and creates an ordered list of chained steps to complete. The end result is a new operating system available to install.
```

Comment:
I'm not sure about this paragraph. While it's true for RPM-related steps of Podman build, there may also be other system configuration steps taking place. Can this be reflected somehow in the sentence?